source/include/platform/acfreebsd.h

```diff
@@ -203,6 +203,10 @@
 
 #define DEBUGGER_THREADING  0   /* integrated with DDB */
 
+#ifdef INVARIANTS
```

Comment:
How about something like
#if defined(INVARIANTS) && !defined(ACPI_MUTEX_DEBUG)
instead?

Reply:
This is kernel-only code and there is no easy way to define ``ACPI_MUTEX_DEBUG`` from command line.